src/edu/ucsb/cs56/projects/games/states_quiz/StopWatch.java

```diff
@@ -8,33 +8,61 @@
  * Provides method to start stopwatch.
  *
  * @author Vincent Wang, Veena Chandran
+ * @author Diana Reyes
  */
 public class StopWatch extends JLabel {
 
-    private int seconds = 0;
+    private int time = 0;
+    private String ready = "Ready?";
+    private String set = "Set.";
+    private String go = "GO!";
+    private int pretime = 0;
 
+
+ 
     public StopWatch(int x, int y, int width, int height) {
         super();
         setBounds(x, y, width, height);
-        setText("Time Elapsed: 00 s");
+
         setVisible(true);
-    }
+
+           }
 
     /**
-     * Starts the stopwatch!
+     * Starts the stopwatch
      */
 
     public void start() {
-        ActionListener task = evt -> {
-            seconds++;
-            setText("Time Elapsed: " + seconds + " s");
+
+            ActionListener task = evt -> {
+            int minutes = time/60;
+	    int seconds = time%60;
+	    
+	    time = time + 1;
+
+	    
+	    
+	    if (seconds <= 9){
```

Comment:
You could do this better.  Hint: [String#format()](https://docs.oracle.com/javase/9/docs/api/java/lang/String.html#format-java.lang.String-java.lang.Object...-)